app/models/history.rb

```diff
@@ -374,17 +374,17 @@ def self.assigned_user(history, create: true)
     create_history(history[:patient], history[:created_by], HISTORY_TYPES[:monitoring_change], compose_message(history, field), create: create)
   end
 
-  def self.pause_notifications(history, create: true)
+  def self.pause_notifications(history)
```

Comment:
I'm a little unclear on why this is being removed? It looks like @jkufro added this just two weeks ago in https://github.com/SaraAlert/SaraAlert/pull/1000.

Reply:
Ah, that would be my mistake. Rebasing this PR was particularly finnicky, and from the look of the conflicts I thought that line was old code that was being persisted on my branch and needed to be removed. I'll restore it asap